Description: don't install dependencies during build
Author: Xavier Guimard <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2021-09-22

--- a/Makefile
+++ b/Makefile
@@ -32,7 +32,7 @@
 
 # don't regenerate the snapshot if we're generating
 # snapshots, since presumably we just did that.
-mandocs: dev-deps $(mandocs)
+mandocs: $(mandocs)
 	@ ! [ $${npm_lifecycle_event} = "snap" ] && \
 	  ! [ $${npm_lifecycle_event} = "postsnap" ] && \
 	  TAP_SNAPSHOT=1 node test/lib/utils/config/definitions.js || true
