Description: fix test
Author: Yadd <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2021-10-06

--- a/test/lib/edit.js
+++ b/test/lib/edit.js
@@ -52,11 +52,11 @@
     editorOpts = null
   })
 
-  return edit.exec(['semver'], (err) => {
+  return edit.exec(['init-package-json'], (err) => {
     if (err)
       throw err
 
-    const path = resolve(__dirname, '../../node_modules/semver')
+    const path = resolve(__dirname, '../../node_modules/init-package-json')
     t.strictSame(editorBin, EDITOR, 'used the correct editor')
     t.strictSame(editorArgs, [path], 'edited the correct directory')
     t.strictSame(editorOpts, { stdio: 'inherit' }, 'passed the correct opts')
@@ -75,8 +75,8 @@
   })
 
   rebuildFail = new Error('test error')
-  return edit.exec(['semver'], (err) => {
-    const path = resolve(__dirname, '../../node_modules/semver')
+  return edit.exec(['init-package-json'], (err) => {
+    const path = resolve(__dirname, '../../node_modules/init-package-json')
     t.strictSame(editorBin, EDITOR, 'used the correct editor')
     t.strictSame(editorArgs, [path], 'edited the correct directory')
     t.strictSame(editorOpts, { stdio: 'inherit' }, 'passed the correct opts')
@@ -96,11 +96,11 @@
     EDITOR = 'vim'
   })
 
-  return edit.exec(['semver'], (err) => {
+  return edit.exec(['init-package-json'], (err) => {
     if (err)
       throw err
 
-    const path = resolve(__dirname, '../../node_modules/semver')
+    const path = resolve(__dirname, '../../node_modules/init-package-json')
     t.strictSame(editorBin, 'code', 'used the correct editor')
     t.strictSame(editorArgs, ['-w', path], 'edited the correct directory, keeping flags')
     t.strictSame(editorOpts, { stdio: 'inherit' }, 'passed the correct opts')
@@ -125,7 +125,7 @@
     gracefulFs.lstat = _lstat
   })
 
-  return edit.exec(['semver'], (err) => {
+  return edit.exec(['init-package-json'], (err) => {
     t.match(err, /lstat failed/, 'user received correct error')
     t.end()
   })
@@ -137,7 +137,7 @@
     EDITOR_CODE = 0
   })
 
-  return edit.exec(['semver'], (err) => {
+  return edit.exec(['init-package-json'], (err) => {
     t.match(err, /exited with code: 137/, 'user received correct error')
     t.end()
   })
